lib/resources/r4/patient_patch.yaml

```diff
@@ -0,0 +1,760 @@
+---
+name: Patient
+field_name_base_url: https://hl7.org/fhir/R4/patient-definitions.html#Patient
+operations:
+- name: add-identifier
+  path: /identifier/-
+  operation: add
+  cardinality: 1..1
```

Comment:
I think the cardinality for all these should actually be 0..*

all the operations can be provided many times, and all are mostly optional right? some tests are required when other ops are present

Reply:
So cardinality here is for the value in the patch operation and not the actual operation.
Might make it more clearer if the headers are updated to `Value Cardinality` and `Value Type`.

Reply:
Updated the headers with https://github.com/cerner/fhir.cerner.com/pull/242/commits/ccc35fc82a441599bcc1e8f2333a0a2324293ef8